### ISSUE TITLE
Update UI of onboarding pages 

### ISSUE THREAD

**Issue by paolodamico** [PostHog/posthog#6022]
Following #5860, let's update the onboarding pages too (I'll be implementing this together): [the preflight page](https://storybook.posthog.net/?path=/story/posthog-onboarding-1-preflight--initial) and the [ingestion modal](https://storybook.posthog.net/?path=/story/posthog-onboarding-3-ingestion--initial).

**Comment by clarkus** [comment]
related https://github.com/PostHog/posthog/issues/5836

**Comment by mariusandra** [comment]
Regarding ingestion. This probably out of scope for the planned work, but sharing the idea regardless.

I recently set up a personal PH instance just to test the charts, and felt that this "ingestion" scene has not kept up with the times, now that we have plugins. Events no longer come in to posthog with just integrations. As an example, all the events in my instance currently come from the "github star sync" plugin. I can imagine enabling other import plugins that'll trickle down events over time. None of this fits with the current ingestion flow that's very integration-heavy.

**Comment by paolodamico** [comment]
Indeed might be a stretch to get this in during the sprint but a super valid point, I'll let @clarkus decide how best to proceed.

**Comment by clarkus** [comment]
I am aiming for parity with the current screens, and then I was going to explore what all it'd take to incorporate plugins. I just want to timebox the work so that I can ensure other projects make progress, too. Regardless, if I don't get to it in this issue I will file a followup enhancement issue.

**Comment by paolodamico** [comment]
Makes sense @clarkus! I would suggest seeing if you can get somewhere with the plugins thing today and if not we can just keep it as a future improvement, but 100% on having the current functionality first

**Comment by clarkus** [comment]
The latest onboarding designs are ready for review at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=3678%3A28162. This flow captures everything we have now in production. I am just now starting on the plugins-based ingestion flow. I've left some guiding comments in figma, but don't feel a need to respond to those. Feel free to comment on anything you see! I'll post another update once plugins are ready for a review.

**Comment by clarkus** [comment]
Plugins ingestion idea is at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=3865%3A27830. This needs more work to illustration the configuration flow, but it's mostly there unless there are other constraints I am missing.

![Ingestion - 03 - Plugins](https://user-images.githubusercontent.com/254612/135164657-13a5cf2c-4732-4892-8fb0-8acf6caea894.png)

**Comment by paolodamico** [comment]
Minor comments added on Figma, but mostly think we're good to go. Thanks @clarkus!

**Comment by clarkus** [comment]
@kpthatsme brought up some good points in figma at https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf?node-id=3865:27830#112486061. It might not be worth spending too much time on the ingestion portion until we have some more improvements to content and workflow figured out.

**Comment by paolodamico** [comment]
Duplicated!
